db/migrations/20190318000000_create_spot_file.sql

```diff
@@ -16,6 +16,20 @@ CREATE INDEX spot_file_mat_header_index
 CREATE INDEX spot_file_mat_ilk_index
     ON maker.spot_file_mat (ilk_id);
 
+CREATE TABLE maker.spot_file_par
+(
+    id        SERIAL PRIMARY KEY,
+    header_id INTEGER NOT NULL REFERENCES headers (id) ON DELETE CASCADE,
+    log_id    BIGINT  NOT NULL REFERENCES header_sync_logs (id) ON DELETE CASCADE,
```

Comment:
Postgraphile basically asks for an index on every FK. So if we want to indulge it, we'll need one on `log_id`.